app/assets/javascripts/hyrax/browse_everything.js

```diff
@@ -11,7 +11,7 @@ https://github.com/samvera/hyrax/blob/v2.3.0/app/assets/javascripts/hyrax/browse
 //= require jquery.treetable
 //= require browse_everything/behavior
 
-$(document).on('ready turbolinks:load', function() {
+Blacklight.onLoad(function() {
```

Comment:
I'm not sure about this one. I think we should keep this one to `$(document).on('ready turbolinks:load',` It looks like @revgum already tried setting it to `Blacklight.onLoad` in https://github.com/osulp/Scholars-Archive/pull/1804 but then changed it to `ready turbolinks:load` for some reason in https://github.com/osulp/Scholars-Archive/pull/1806 to address issue https://github.com/osulp/Scholars-Archive/issues/1802

Reply:
Hrm. Ill remove this, but yeah that could also be a part of why large files arent uploading proper. Ill change this back.